script/Main.lua

```diff
@@ -3,13 +3,13 @@ package.path = package.path .. ';./script/?.lua'
 package.path = package.path .. ';./script/?.ext.lua'
 package.path = package.path .. ';./script/?.ffi.lua'
 
----@type EngineImpl
```

Comment:
i´d get rid of those and below, you can use the SetEngine i posted

Reply:
Are these still useful for linting purposes, so the LLS knows that these are of type Engine or whatever?